client/src/pages/LandingPage.tsx

```
@@ -31,151 +33,141 @@ export default function LandingPage() {
+              setStreamingStatus(
+                `Detected: ${parsedData.data} project - Generating code...`
+              );
+              break;
+
+            case "code":
+              accumulatedResponse += parsedData.data;
+              setStreamedResponse(accumulatedResponse);
+
+              const fileMatch = accumulatedResponse.match(
+                /<boltAction[^>]*type="file"[^>]*>([\s\S]*?)<\/boltAction>/i
+              );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)